Moggles/ClientApp/TogglesList.vue

```diff
@@ -465,26 +557,24 @@
                         });
 						return rowModel;
 					});
-
                     this.toggles = gridRowModels;
 					Bus.$emit('toggles-loaded', gridRowModels);
-
 				}).catch(() => {
 					//do not uncomment this, the null reference exception will return to haunt us !
 					//window.alert(error)
 				});
 			},
 			initializeGrid(app) {
-				this.environmentsList = [];
-
+                this.environmentsList = [];
+                Bus.$emit('toggle-scheduled');
```

Comment:
I think you should call getAllScheduledToggle directly without emitting the event even if they do the same thing. Maybe in the future the handler will be modified and it will not apply for this case anymore and because also the action performed is not resulted after a toggle was scheduled